### ISSUE TITLE
Start and end of secretion

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#16471]
From #11255

@vanaukenk raises these important points for secretion - we need to clarify: 

Wrt the term definition, what are the key things we want to capture?
    That the secretion is regulated? If so, by what?
    That the secretion is accompanied by peptide processing?
    That the secretion acts upon a particular class of molecule?
    That the secretion happens in particular cell types?
    That the secretory vesicles share similar structural/functional features?

**Comment by ukemi** [comment]
Secretion can also be mediated by a tissue or organ can't it? What if an organ stores a substance and then secretes it by contracting?

**Comment by RLovering** [comment]
Hi

sorry I put this on the wrong tracker  Definition for secretion term #2738 

please could the definition for secretion be improved to provide a beginning and end to this process. Currently it is not clear that this term does not include the loading of a vesicle etc.

GO:0046903 secretion
current definition: The controlled release of a substance by a cell or a tissue.

Suggested definition: The controlled release of a substance by a cell or a tissue. The process starts when the substance to be released is localised at the release site and ends when the substance leaves the cell or tissue.

Possibly add the note: loading of the substance within a secretory vesicle is not included in the secretory process as this is covered by the process of GO:0035459 vesicle cargo loading.

Also possibly add this clarification to some of the child terms.
eg:
GO:0032940 secretion by cell
Suggested definition: The controlled release of a substance by a cell. The process starts when the substance to be released is localised at the release site and ends when the substance leaves the cell.

If people feel that the definition of secretion should include earlier processes then it would be helpful to have some new child GO terms associated with this term to clarify this, as well as changes to the definition.

Thanks

Ruth

**Comment by RLovering** [comment]
comment from Pascale
HI @RLovering

I like your suggestion. Many signaling proteins are annotated to 'secretion', while I think it would be more accurate to annotate them to 'regulation of secretion'. Specifying the beginning would be very useful here.

Thanks, Pascale

**Comment by pgaudet** [comment]
@ukemi I assume we agree with the new def ? Should we have a matrix rule to check secretion x signaling ?

**Comment by ValWood** [comment]
secretion is a tricky term to describe starts and ends.

are you referring to secretion from a single cell  or a tissue.

We have this term
GO:0046903    secretion
The controlled release of a substance by a cell or a tissue.

and
GO:0032940    secretion by cell
The controlled release of a substance by a cell.

how does secretion by a cell differ from regulated endocytosis?
GO:0045055    regulated exocytosis
A process of exocytosis in which soluble proteins and other substances are initially stored in secretory vesicles for later release. It is found mainly in cells that are specialized for secreting products such as hormones, neurotransmitters, or digestive enzymes rapidly on demand.

?

and then:
GO:1990182    exosomal secretion
Definition (GO:1990182 GONUTS page)
The process whereby a membrane-bounded vesicle is released into the extracellular region by fusion of the limiting endosomal membrane of a multivesicular body with the plasma membrane. PMID:10572093 PMID:12154376 PMID:16773132 PMID:18617898

I'm not sure why this term. A lot of these sound the same. It is possibly because  pheromones are sometimes  'secreted' by transmembrane transporters?


Just to note the new definition needs to work for all of the descendants. I think it is probab ly OK to say the cargo is already loaded, if you are talking about exosomal secretion.

Also, despite a huge number of annotation secretion has only 14  direct annotations.
Do we actually require the term secretion to refer to secretion from a tissue,  since we are annotating genes aren't we always talking about secretion from cells?

Just some things I often wondered....

**Comment by ValWood** [comment]
For example, we have

transporters annotated under 'secretion' 
abcb11b | ATP-binding cassette, sub-family B (MDR/TAP), member 11b |   | bile acid secretion

which would not fit the proposed def.


-- | -- | -- | --

**Comment by RLovering** [comment]
Hi Val
do you mean Suggested definition: The controlled release of a substance by a cell or a tissue. The process starts when the substance to be released is localised at the release site and ends when the substance leaves the cell or tissue.
I don't see what the problem is with transporters and this definition, it doesn't exclude the role of the transporter earlier in the process.
Ruth

**Comment by ValWood** [comment]
It might be OK. I just thought that "localised at the release site" seems to apply specifically to vesicles.

**Comment by raymond91125** [comment]
I thought secretion is like dam release -- its start and end (duration) is being controlled.

The current definition does not distinguish secretion from excretion. In fact, isn't excretion a type of secretion?

**Comment by ValWood** [comment]
>I thought secretion is like dam release -- its start and end (duration) is being controlled.

I agree, at least this is my understanding at the single-cell level. I suspect a lot of the problems are that "secretion" is conflated to include cell-level and tissue-level secretion (although I am not sure how tissue level secretion is annotated in a gene-specific way, this is outside my experience).

A search of the tracker for the term secretion reveals  28 open and 589 closed tickets which usually indicates we are getting something really wrong here...
We  have many terms containing "secretion" ranging from "tear secretion", to specific amino acid secretion "glycine secretion" (one IMP annotation MGI to a  transmembrane transporter, is this really secretion?), mucus secretion, saliva secretion, various hormone secretion, "signal release", and then specific terms for signal release form specific cell types (i.e synapse). 

So define  the current use of 'secretion' in GO is almost impossible because it is used in different contexts (i.e. cell and tissue level)

**Comment by ValWood** [comment]
The start and end would need to be *very broad* for secretion.

<img width="260" alt="Screenshot 2021-07-23 at 21 21 30" src="https://user-images.githubusercontent.com/7359272/126837399-e1555cbc-ae19-4d5b-a1b5-d45b230452d6.png">

In single-celled organisms we consider  'secretion' as the 'cell level secretory system' beginning with import into the ER and ending once a protein is externalised. 

However, yeast pheromones are exported via transporters, not via the secretory system. Is the same true of higher eukaryotic hormones/pheromones? i.e. is  "tissue level secretion" sometimes via the ' cell level secretory system', and sometimes by a high density of  transmembrane transporters, rather than  by the the 'vesicle-mediated secretory system' directly ?
Obviously, both processes will be involved, even in the TM transport situation because v-m-t is required to get the transporters to the cell surface. Therefore, secretory system mutants will inhibit tissue level secretion indirectly because the same system is used to localize cell surface transporters as to do endocytosis.

If so, the logic problem seems to be:
Secretion is exporting some substances from cells. 
All endocytosis is secretion.

BUT
Only some export "GO:0140115   export across plasma membrane" (defined as transmembrane transport) is secretion (i.e in multicellular tissues)
For instance, in yeast  inorganic compound export across the plasma membrane in yeast would not be considered as secretion.
 
I think if you can satisfy these criteria it should be possible to curate a beginning and end.
v

**Comment by ValWood** [comment]
> In fact, isn't excretion a type of secretion?

I don't think so. Excretion  is defined as system-level processes 

e.g 
defecation
The expulsion of feces from the rectum.

we don't want to get into that shit here ;)

**Comment by mah11** [comment]
fixed critical typo in [this comment](https://github.com/geneontology/go-ontology/issues/16471#issuecomment-885907654)

In the Olden Days GO made an effort to distinguish the messy & varied ways researchers use "secretion", with the high-level 'secretion by cell' / 'secretion by tissue' split. The subclasses of the latter look like that's where they belong as long as you want to keep them at all. But there could well be scope to improve definitions, and the subclasses of 'secretion by cell' look like a grab bag. I can't say much beyond that.
